### ISSUE TITLE
"Migrating to Up from Now"

### ISSUE THREAD

**Issue by j** [apex/up#732]
(not a real issue)

We've been users of Up for a while now and have used all of the current lambda deployment implementations and Up has always treated us the best.

With that said, Now v2 was just launched and none of our apps work that are on it (because they were either Docker deploys or their new version just doesn't work because it does too much magic).

It's a good time to maybe write a blog post about migrating for many reasons.  Up is cheaper, works, and doesn't have the same current v2 limits.

Unrelated:
  - When are other events (SNS mainly) going to be supported if ever?

Thanks, @tj.  Been following you for years (esp back in your node days :P)

**Comment by komuw** [comment]
a lot of people seem frustrated with `now` v2 -> https://news.ycombinator.com/item?id=18407503

**Comment by j** [comment]
Yup, exactly. I enjoyed zeits stuff and was lambda is a comforting move. We had some stuff in production that went down many times and came running back to Up.  We want to use only up but use serverless for SNS based services.

**Comment by tj** [comment]
Cool cool! Yeah I'll have to give the other FaaS stuff some thought, HTTP/apps is definitely only one tiny portion of this stuff, SQS, Kinesis, SNS, all kinds of awesome things to tap into on AWS. Sounds like they've kind of ruined their niche, in exchange for a tiny portion of what makes FaaS great

**Comment by tj** [comment]
I didn't realize they were literally repurposing AWS Lambda as well hahah.... I was planning on repurposing pretty much everything in AWS but they're definitely going that route now. Not sure it makes sense to compete with a startup doing the same thing though, it's much harder to afford it without all that free $$, not making much as-is. Tired of all these ridiculous entitled startups, same old shit, make OSS, sell your team, pretend you're sweet

**Comment by j** [comment]
@tj no s**t huh, lol.  They really lost their user-base.  I mean, "Serverless" is already a thing.  Up is already a thing.  I don't see why they try to compete with Serverless when their first iteration has no support for anything BUT http events.  The docker deployments were their bread and butter.  In fact, I had apps that were too complex for Lambda and started going the Now + Docker route and next day the v2 announcement happened and I scrapped it.  Their Cloudflare partnership was pretty cool though.

**Comment by siegesmund** [comment]
@tj, one big difference between now v2 and Up is that now v2 pushes developers to decompose each route into an individual lambda, a la the Serverless framework as I understand it. In this way, it looks more like Serverless than Up to me.
